src/main/java/org/veupathdb/service/eda/ds/plugin/filteredmetadata/ContinuousVariablePlugin.java

```diff
@@ -88,13 +88,19 @@ protected void writeResults(OutputStream out, Map<String, InputStream> dataStrea
 
           PluginUtil util = getUtil();
           ContinuousVariableMetadataSpec spec = getPluginSpec();
-          List<String> requestedMetadata = spec.getMetadata();    
+          List<String> requestedMetadata = spec.getMetadata();
 
           useRConnectionWithRemoteFiles(Resources.RSERVE_URL, dataStreams, connection -> {
             connection.voidEval(util.getVoidEvalFreadCommand(DEFAULT_SINGLE_STREAM_NAME, spec.getVariable()));
             // for convenience
             connection.voidEval("x <- " + DEFAULT_SINGLE_STREAM_NAME + "$" + util.toColNameOrEmpty(spec.getVariable()));
 
+            // Dates are read in as class=character. If we see the variable type is date, let's
+            // force the data in R to be have class=date. See veupathUtils #14 for more details.
+            if (util.getVariableType(spec.getVariable()) == "DATE") {
```

Comment:
There's probably a better way to check this using like APIVariableType or something